plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -808,15 +810,26 @@ test.describe( 'Add New Variable Product Page', () => {
 			'#variable_product_options .toolbar-top a.expand_all'
 		);
 		await page.check( 'input.checkbox.variable_manage_stock' );
-		await page.fill( 'input#variable_regular_price_0', variationOnePrice );
+
+		const firstVariationContainerLocator = await page
+			.locator( '.woocommerce_variations  .woocommerce_variation' )
+			.first();
+
+		await firstVariationContainerLocator
+			.getByPlaceholder( 'Variation price (required)' )
+			.fill( variationOnePrice );
 		await expect(
 			page.locator( 'p.variable_stock_status' )
 		).not.toBeVisible();
-		await page.fill( 'input#variable_stock0', stockAmount );
+		await firstVariationContainerLocator
+			.getByLabel( 'Stock quantity' )
+			.fill( stockAmount );
```

Comment:
The tests are failing when we run this because this locator resolves to 3 elements.  We'll have to get a little more specific here (maybe add a `.first()` before the `.fill()`?

Reply:
@lanej0 All tests inside this file pass without any change.

```
USE_WP_ENV=1 pnpm playwright test e2e-pw/tests/merchant/create-variable-product.spec.js --config=tests/e2e-pw/playwright.config.js
```

<img width="1057" alt="image" src="https://user-images.githubusercontent.com/1784821/233251735-5f98ee23-185e-4757-8265-a76c8a96b5f0.png">

Reply:
The failing test count is zero when running all tests.

<img width="1064" alt="image" src="https://user-images.githubusercontent.com/1784821/233254296-420978e2-ff96-4e08-b64b-abc75c3a27ea.png">